pkgs/by-name/ve/verible/package.nix

```diff
@@ -53,7 +53,7 @@ buildBazelPackage rec {
       {
         aarch64-linux = "sha256-jgh+wEqZba30MODmgmPoQn1ErNmm40d16jB/kE2jYPg=";
         x86_64-linux = "sha256-kiI/LX0l9ERxItsqiAyl+BP3QnLr0Ly2YVb988M4jVs=";
-        aarch64-darwin = "sha256-bkw4ErWYblzr3lQhoXSBqIBHjXzhZHeTKdT0E/YsiFQ=";
+        aarch64-darwin = "sha256-Ivx6KCzAxy6DAffVirb90ZMSGTIS/UXQEhZQebFjV3s=";
```

Comment:
```
error: hash mismatch in fixed-output derivation '/nix/store/9nkz0g7xcppsz1dfdhn4s2z35z9qz671-verible-0.0.3956-deps.tar.gz.drv':
           likely URL: (unknown)
            specified: sha256-Ivx6KCzAxy6DAffVirb90ZMSGTIS/UXQEhZQebFjV3s=
                  got: sha256-bkw4ErWYblzr3lQhoXSBqIBHjXzhZHeTKdT0E/YsiFQ=
        expected path: /nix/store/d30pzvj08p7hlhsb3q4fczhp790cmyn7-verible-0.0.3956-deps.tar.gz
             got path: /nix/store/0d8mncfgdyppca3vgchwkg0qj30lfx2i-verible-0.0.3956-deps.tar.gz
```

This is most likely a case-sensitivity issue – please don’t break the hash for users with case‐sensitive stores. It should be normalized in `postFetch` instead.

Reply:
Ok, good to know. How exactly should this be done? Is there any related section in the manual?

Reply:
Just a matter of figuring out what files are colliding in case/normalization, figuring out if that’s load‐bearing, and then adjusting `postFetch` to create a result without such collisions. Then the hash is the same regardless.